### ISSUE TITLE
wrong handling of nginx access logs in webserver container

### ISSUE THREAD

**Issue by PetrHeinz** [shopsys/shopsys#1326]
### What is happening

There is a directive `access_log /var/log/nginx/shopsys-framework.access.log` in our [nginx.conf](https://github.com/shopsys/shopsys/blob/v7.3.1/project-base/docker/nginx/nginx.conf) configuration. This config is used for running nginx in Docker and in containerized applications we rather use `stdout` for logs (see [12-factor app](https://12factor.net/logs)).

This means there will be a log file in the `webserver` with duplicate contents with logs maintained by Docker itself (as it reads and stores `stdout` from all containers). This logfile can get pretty big pretty soon...

### Expected result

The nginx in container should be configured to not save logs into a file. Let Docker (or Kubernetes) handle the logging.

**Comment by PetrHeinz** [comment]
I've noticed the two symlinks `access.log -> /dev/stdout` and `error.log -> /dev/stderr` in `/var/log/nginx/`:

```
# ls -lah /var/log/nginx/
total 646016
drwxr-xr-x    1 root     root          42 Aug 15 05:16 .
drwxr-xr-x    1 root     root          19 Apr 10  2018 ..
lrwxrwxrwx    1 root     root          11 Apr 10  2018 access.log -> /dev/stdout
lrwxrwxrwx    1 root     root          11 Apr 10  2018 error.log -> /dev/stderr
-rw-r--r--    1 root     root      624.4M Aug 16 08:11 shopsys-framework.access.log
```

Seems to me that specifying the `access_log` directive in `nginx.conf` just disabled streaming the access logs to `stdout`, so the docker log doesn't really contain duplicate contents as I originally wrote in the description. It contains the error log only.

Hard to know whether that was intentional. The directive was introduced quite a while ago in 375a272a28090ab7a9d5d60c3b5a38f9c310ce19. @TomasLudvik @vitek-rostislav Do you remember why it was added?

I've tested the removal of the `access_logs` directive and it works as expected. After the removal, access logs were accessible via `docker-compose logs webserver`. With the directive in place, only error logs were accessible.

**Comment by PetrHeinz** [comment]
The logs can get large, for that reason we could introduce some logging options to the `docker-compose.yml` to configure log rotation:

```diff
  webserver:
      image: nginx:1.13-alpine
      container_name: shopsys-framework-webserver
      depends_on:
          - php-fpm
      volumes:
          - .:/var/www/html
          - ./docker/nginx/nginx.conf:/etc/nginx/conf.d/default.conf
      ports:
          - "8000:8080"
+     logging:
+          options:
+              max-size: "10m"
+              max-file: "10"
```

This example config will keep the logs of the `webserver` container at the maximum size of 100 MiB.

See [Compose file reference](https://docs.docker.com/compose/compose-file/#logging) and [Docker logging drivers configuration docs](https://docs.docker.com/config/containers/logging/configure/) for more details.

**Comment by TomasLudvik** [comment]
@PetrHeinz that was default nginx.conf used by Shopsys in that time :)

**Comment by PetrHeinz** [comment]
Fixed in #1362
